examples/fake-ad.amp.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
```

Comment:
actually, would you move this file to `test/manual/fakeada4a.amp.html`, to sit together with `test/manual/fakead3p.amp.html`